python_src/src/lamp_py/ingestion/pipeline.py

```diff
@@ -81,6 +85,11 @@ def start() -> None:
         validate_db=True,
     )
 
+    # run metadata rds migrations
+    alembic_upgrade_to_head(db_name="metadata_prod")
+
```

Comment:
migrations for the metadata db are owned by ingestion. this should make our deployment experience better.

Reply:
This does complicate local development, since we don't have/don't really want to run a local ingestion image, standing up the local metadata rds would need a different configuration.

One option would be to allow the 'seed_metadata` image to do this, but it would require some re-working of that logic (probably some different env vars)

Reply:
good note. lets discuss in person, but i think adding to `seed_metadata` makes sense.